### ISSUE TITLE
SystemStackError when creating page

### ISSUE THREAD

**Issue by parndt** [refinery/refinerycms#2847]
This is caused by refinerycms-wymeditor version 1.0.4.. not sure quite *why* yet.  It's most likely caused by the decorators library.

**Comment by CJBrew** [comment]
There's another issue I'm seeing, on doing just about anything involving either `rails` or `rake`:

```shell
Bundler is using a binstub that was created for a different gem.
This is deprecated, in future versions you may need to `bundle binstub refinerycms-wymeditor` to work around a system/bundle conflict.
```

Any idea what might be causing that?

**Comment by fixnum** [comment]
It seems that `friendly_id_options_decorator.rb` in refinerycms-wymeditor gem is the culprit.

**Comment by parndt** [comment]
I have no idea why Bundler is doing that but I'm facing the same issue!

**Comment by CJBrew** [comment]
Rebuilding the binstubs (as suggested on a number of Stackoverflow or similar posts) has no effect. 

I am wondering whether it's some sort of versioning issue that's confusing Bundler. 

i.e.:
```Gemfile
gem 'refinerycms-wymeditor', ['~> 1.0', '>= 1.0.4']
```

Why two declarations of the Gem version? Maybe it's right, I don't know...

**Comment by parndt** [comment]
No that declaration is fairly ordinary unfortunately

**Comment by CJBrew** [comment]
Don't know if this will be of any use...
https://github.com/bundler/bundler/issues/2928#issuecomment-38292479

**Comment by CJBrew** [comment]
Note: my pull request fixes the binstubs issue, not the stack error.

**Comment by CJBrew** [comment]
@fixnum confirming your change appears to fix the Stack Error for me. Thanks.

**Comment by CJBrew** [comment]
@parndt  Sorry, this shouldn't be closed. My fix doesn't correct the stack error issue.

I can send a PR of @fixnum's change -- I pulled that to my fork for testing -- but he may prefer to do it.

**Comment by parndt** [comment]
Oh, right, of course.. sorry about that.

**Comment by CJBrew** [comment]
PR sent

**Comment by parndt** [comment]
This will be fixed by #2855
